test/lib/ufe/testTransform3dChainOfResponsibility.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2020 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import maya.cmds as cmds
+
+import mayaUsd.ufe
+
+import mayaUtils
+import ufe
+
+from pxr import UsdGeom, Vt
+
+import unittest
+
+class Transform3dChainOfResponsibilityTestCase(unittest.TestCase):
```

Comment:
This is great. Do we have a way to change the chain of responsibility from python at runtime?

Reply:
No, no bindings.